eng/Tests.ruleset

```diff
@@ -0,0 +1,22 @@
+<RuleSet Name="Tests ruleset" Description="All Rules are disabled." ToolsVersion="15.0">
```

Comment:
This is something that xunit knows to interpret/use?

Reply:
No, this is something Roslyn understands. 

The newest version of xunit brings in analyzers as well. This is the way to configure roslyn analyzers. Then you can set the severity to error, warning or none, on a per rule, per analyzer basis. So whenever we want to enable these rules, we will have to fix the errors we get by the analyzer.

Reply:
It was just a smaller change to do this and  then we can enable the rules by chunks.

Reply:
I see. So when we want to turn on one of these rules, we change `Action="None"` -> say `Action="Error"(or Warning)`?

Reply:
Yeah, or if you don't list them in this file they will take whatever default Action the analyzer defines: http://xunit.net/xunit.analyzers/rules/